container/pull.bzl

```diff
@@ -144,7 +150,11 @@ def _impl(repository_ctx):
 
     puller = repository_ctx.attr.puller_linux_amd64
     if repository_ctx.os.name.lower().startswith("mac os"):
-        puller = repository_ctx.attr.puller_darwin
+       arch = repository_ctx.execute(["uname", "-m"]).stdout.strip()
```

Comment:
We should use https://docs.bazel.build/versions/main/skylark/lib/repository_os.html instead.